### PR TITLE
Fix the TS table field mappings

### PR DESCRIPTION
**Pull request by elic-eon**
### 🤔 Why?

The `physicalColumnGUID` is not available now, the lineage between "table" and external source table is broken

### 🤓 What?

- Use the connection to get the source entity id

### 🧪 Tested?

Update tests. Verified the MCEs before and after are identical.

### CODE REVIEW

**Review by mars-lan — COMMENTED** [review]
Bump up version?

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "metaphor-connectors"
-version = "0.11.93"
```
Comment (mars-lan) [inline pyproject.toml:3]: Bump up again.

**Review by mars-lan — COMMENTED** [review]
Otherwise LGTM

File: pyproject.toml (hunk 1 above)
Comment (elic-eon) [reply pyproject.toml]: Updated

**Review by mars-lan — APPROVED** [review]

**Pull request merged**